opencart 1.5.6.4/upload/admin/controller/module/smaily_for_opencart.php

```diff
@@ -99,6 +101,30 @@ public function index() {
         $this->data['rss_feed_title'] = $this->language->get('rss_feed_title');
         $this->data['rss_feed_text'] = $this->language->get('rss_feed_text');
         $this->data['rss_feed_url'] = $this->config->get('config_url') . 'index.php?route=smailyforopencart/rss';
+
+        // Layout text and buttons
+        $this->data['entry_layout'] = $this->language->get('entry_layout');
```

Comment:
What does `entry` stand for?

Reply:
OK. I see, because OpenCart provides the phrase out of the box. Keep it as it is then.